src/lwc/geBatchGiftEntryHeader/geBatchGiftEntryHeader.js

```diff
@@ -17,6 +18,8 @@ export default class GeBatchGiftEntryHeader extends LightningElement {
     @api batchTotals = {};
     @api isPermissionError;
 
+    isElevateCustomer;
```

Comment:
Can we create a getter that returns the value of `isElevateCustomer` so that's referenced in the markup?

Reply:
An alternative would be to piggyback off of the parent component's existing connectedCallback and pass the `isElevateCustomer` value into the header component via a public prop.